docs/workstream-governance.md

```diff
@@ -0,0 +1,35 @@
+# Software Supply Chain Workstreams
+
+## Overview and Motivations
+
+Project and user representatives who take part in the Software Supply Chain SIG are welcome to form a workstream to work on specific areas of interest in a more focused and structured way.
+
+The expectations from workstreams would vary depending on the area of focus. Workstreams could focus on collecting the state of things in a specific area from within the ecosystem and contribute that to the domain, or come up with a proposal to address some of the needs in a more structured way in order to socialize their ideas further and increase the awareness.
+
+Workstreams should, at a regular cadence, share their progress with the rest of the SIG. Workstreams could liase with the Software Supply Chain SIG to present ideas per area on CDF TOC level in order to gather feedback.
+
+## Lifecycle
+
+Working groups are temporary in nature. They are formed to focus on specific goals and can be disbanded after reaching their goals. Upon formation, leads for the working group are choosen by the working group. When leads step down, they can be replaced by [lazy concensus](https://wiki.openoffice.org/wiki/Documentation/FAQ/ProjectLevel/CommunityQuestions/What_is_%22Lazy_Consensus%22%3F). However, working groups can also be disbanded if there is no longer a lead for the working group (within an 8 week grace period).
```

Comment:
suggest "Workstreams are temporary..." for consistency (rather than introducing a "working group" concept here)

similarly: "leads for the sowrkstream are chosen by the participants"

Reply:
Thanks, updated!